reflex/components/radix/themes/components/context_menu.py

```diff
@@ -232,8 +233,26 @@ class ContextMenuSeparator(RadixThemesComponent):
     tag = "ContextMenu.Separator"
 
 
+class ContextMenuCheckbox(RadixThemesComponent):
```

Comment:
I think if this inherit from HighLevelCheckbox, we won't need to redefined the `create` method, and the docs will have all the props of the checkbox when we generate them.

Reply:
I tried leaving it out, but Reflex doesn't recognize the `on_change` event trigger even though it was already defined in `checkbox.py`. I also saw how the `separator` was used in `context_menu.py`, but didn't see its imports. Could you give me some insights?

Reply:
If you define it as `class ContextMenuCheckbox(RadixThemesComponent)`:
it means -> `on_change` is not known because you haven't defined it and `RadixThemesComponent` doesn't define it either

If you write as `class ContextMenuCheckbox(HighLevelCheckbox)`
it means -> `on_change` should be known because it is defined in `HighLevelCheckbox`

Reply:
Thank you!